packages/circuit-ui/components/TopNavigation/TopNavigation.tsx

```diff
@@ -44,10 +40,6 @@ export interface TopNavigationProps
     HTMLAttributes<HTMLElement> {
   logo: ReactNode;
   hamburger?: HamburgerProps;
-  /**
-   * @deprecated Use a custom component in the `links` prop instead.
-   */
-  profileMenu?: Omit<ProfileMenuProps, 'user'>;
   /**
    * @deprecated Use a custom component in the `links` prop instead.
    */
```

Comment:
The `user` prop was only used for the profile menu, so it can be removed as well.